python/ray/tests/test_unreconstructable_errors.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import unittest
+
+import ray
+
+
+class TestUnreconstructableErrors(unittest.TestCase):
```

Comment:
Does it make sense to put these in a new file? Maybe they should go in `test_failure.py` instead.

Reply:
Isn't that test file too large already?

Reply:
Hmm I'd prefer we just put it there now and reorganize that whole file later, but up to you.